backends/advanced/src/advanced_omi_backend/services/memory/providers/mycelia.py

```
@@ -0,0 +1,827 @@
+
+        Raises:
+            ValueError: If user not found
+        """
+        from advanced_omi_backend.auth import generate_jwt_for_user
+
+        # If email not provided, lookup user
+        if not user_email:
+            from advanced_omi_backend.users import User
+            user = await User.get(user_id)
+            if not user:
+                raise ValueError(f"User {user_id} not found")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)